codewars22.py

```diff
@@ -0,0 +1,18 @@
+# https://www.codewars.com/kata/53f40dff5f9d31b813000774/train/python
```

Comment:
Hi Piotr, I think this is the wrong Kata linked here. Please input the right link, so I can check it properly. Thanks!

Reply:
You're right! My apologies. Updated the kata url.